src/services/riotApiClient.js

```diff
@@ -10,63 +10,65 @@ class RiotApiClient {
+
+    let response;
+    try {
+      response = await axios.get(url,
+          {
+            baseURL: config.get('riot.baseURL'),
+            headers: {
+              'X-Riot-Token': config.get('riot.apiKey')
             }
-        )
-        return this.transformSummonerInfo(response.data)
-      } catch (err) {
-        logger.error(`An error occurred attempting to fetch from ${response.config.url}, ${err.message}`)
-        throw err
-      }
+          }
+      )
+      return this.transformSummonerInfo(response.data)
+    } catch (err) {
+      logger.error(`An error occurred attempting to fetch from ${response.config.url}, ${err.message}`)
+      throw err
     }
+  }
 
-    async fetchTFTSummonerInfoByPuuid(puuid) {
-      const url = `/tft/summoner/v1/summoners/by-puuid/${puuid}`
-  
-      let response;
-      try {
-        response = await axios.get(url,
-            {
-              baseURL: config.get('riot.baseURL'),
-              headers: {
-                'X-Riot-Token': config.get('riot.apiKey')
-              }
+  async fetchTFTSummonerInfoByPuuid(puuid) {
+    const url = `/tft/summoner/v1/summoners/by-puuid/${puuid}`
+
+    let response;
+    try {
+      response = await axios.get(url,
+          {
+            baseURL: config.get('riot.baseURL'),
+            headers: {
+              'X-Riot-Token': config.get('riot.apiKey')
             }
-        )
-        return this.transformSummonerInfo(response.data)
-    
-      } catch (err) {
-        logger.error(`An error occurred attempting to fetch from ${response.config.url}, ${err.message}`)
-        throw err
-      }
+          }
+      )
+      return this.transformSummonerInfo(response.data)
+  
+    } catch (err) {
+      logger.error(`An error occurred attempting to fetch from ${response.config.url}, ${err.message}`)
+      throw err
     }
+  }
+  
   // Fetches the most up to date version of data
-    async fetchLatestLolPatchVersion() {
-      let response;
-      try {
-      response = await axios.get('https://ddragon.leagueoflegends.com/api/versions.json')
-      } catch (e) {
-          return res.status(500).send(`Unable to fetch Json: ${e.message}`)
-      }
-      const versions = response.data
-      this.latestLolPatchVersion = versions[0]
+  async fetchLatestLolPatchVersion() {
+    let response
+    try {
+    response = await axios.get('https://ddragon.leagueoflegends.com/api/versions.json')
+    } catch (e) {
+        return res.status(500).send(`Unable to fetch Json: ${e.message}`)
+    }
+    const versions = response.data
+    this.latestLolPatchVersion = versions[0]
   
-   }
-   // Fetches whatever amount of most recent match id's you want
-   async getRecentMatches(puuid, count) {
-     const puuidURL = `/tft/match/v1/matches/by-puuid/${puuid}/ids?count=${count}` 
-     let response;
-     try{
+  }
+   
+  // Fetches whatever amount of most recent match id's you want
+  async getRecentMatchesList(puuid, count) {
```

Comment:
Name this function fetchRecentMatches(puuid, count)
```suggestion
  async fetchRecentMatches(puuid, count) {
```